core/src/script/CGXP/plugins/Print.js

```diff
@@ -225,17 +249,88 @@ cgxp.plugins.Print = Ext.extend(gxp.plugins.Tool, {
      *        autoCreate: {tag: "input", type: "text", size: "45", maxLength: "45"}
      *    }]
      *
-     *  Default to: ``['title', 'comment', 'legend']``, that are predefined fields.
+     *  Default to: ``[]``.
+     */
+    fields: [],
+
+    /** api: config[options]
+     *  ``Object``
+     *  The options given to the print panel.
+     *  E.g.:
+     *
+     *  .. code:: javascript
+     *
+     *      options: {
+     *          labelAlign: 'top',
+     *          defaults: {
+     *              anchor:'100%'
+     *          },
+     *          autoFit: true,
+     *          fieldsExtraClientConfiguration: {
+     *              "A4_portrait": {
+     *                  "title": {
+     *                      fieldAttributes: {
+     *                          fieldLabel: "${_("Name")}",
+     *                          emptyText: "${_("Name")}"
+     *                      }
+     *                  },
+     *                  "description": {
+     *                      useTextArea: true,
+     *                      fieldAttributes: {
+     *                          fieldLabel: "${_("Description")}",
+     *                          emptyText: "${_("Description")}"
+     *                      }
+     *                  }
+     *              }
+     *          }
+     *      }
+     *
+     *  Default to: ``{}``.
      */
-    fields: ['title', 'comment', 'legend'],
+    options: {},
+
+    /** api: config[additionalAttributes]
+     *  ``Array``
+     *  Attributes added in the print form used especially with print V2.
+     *
+     *  Default to:
+     *
+     *  .. code:: javascript
+     *
+     *    [{
+     *        name: 'Title',
+     *        type: 'String'
+     *    }, {
+     *        name: 'Comments',
+     *        type: 'String'
+     *    }, {
+     *        name: 'legend',
+     *        type: 'LegendAttributeValues'
+     *    }]
+     */
+    additionalAttributes: [{
+        name: 'title',
+        label: 'Title',
+        type: 'String'
+    }, {
+        name: 'comment',
+        label: 'Comment',
+        type: 'String',
+        useTextArea: true
+    }, {
+        name: 'legend',
+        type: 'LegendAttributeValues'
+    }],
```

Comment:
Oops I see lower in the PR that the maxlength of textfields and textareas are hard-coded to 45 and 100.